lms/services/http.py

```diff
@@ -36,6 +42,38 @@ def request(self, method, url, timeout=(10, 10), **kwargs) -> Response:
         try:
             response = self.session.request(method, url, timeout=timeout, **kwargs)
             response.raise_for_status()
+
+        except requests.exceptions.SSLError:
+
+            class CustomSession(AIASession):
+                def __init__(self):
+                    self.user_agent = DEFAULT_USER_AGENT
+                    self._context = ssl.create_default_context()
+
+                    self._context.check_hostname = False
```

Comment:
Had to add this extra lines to be extra lax while downloading the certificates themselves. I'm not sure how much we are compromising here security wise.